src/main/resources/db/changelog/changes/create-consortium-table.xml

```diff
@@ -17,4 +17,14 @@
 
   </changeSet>
 
+  <changeSet id="MODCON-23@add-metadata-fields" author="singhAdesh">
+    <addColumn tableName="consortium">
+      <column name="created_by" type="UUID"/>
+      <column name="created_date" type="TIMESTAMP WITHOUT TIME ZONE" defaultValueComputed="NOW()">
```

Comment:
lets use lower case to defining types as for previous change sets to have consistent naming

Reply:
fixed